strings.Rmd

```diff
@@ -386,7 +386,7 @@ str_detect(c("grey", "gray"), "gr(?:e|a)y")
 1.  Describe, in words, what these expressions will match:
 
     1. `"(.)(.)\\2\\1"`
-    1. `(..)\1`
+    1. `"(..)\1"`
```

Comment:
I think the lack of quotes there was deliberate, to force you to think about the regular expression vs. the string the represents the regular expression